app/controllers/login_controller.rb

```diff
@@ -23,7 +23,7 @@ def urs
     else
       redirect_to root_path
       if (@user.role.nil?)
-        flash.notice = "User is not provisioned with the proper ACLs.   Please contact the Earthdata Operations team."
+        flash.notice = "The user is not provisioned with the proper ACLs. Please contact User Support at support@earthdata.nasa.gov."
```

Comment:
No test breakages?  Do we have a test for this condition?  If not, I think we should write one.   I think the error is occurring when a dashboard is attempting to retrieve ACLs and fails (least I think that is what is happening).

Reply:
No Test breakages; however just added 1 new test based on the feedback.